CHANGELOG.md

```diff
@@ -46,7 +46,7 @@ _None._
 
 ### Internal Changes
 
-_None._
+- Change the NSObject+SafeExpectations dependency to `~> 0.0.4`. [#555]
```

Comment:
Apologies for not thinking about this sooner, but should we acknowledge that the `NSObject+SafeExpectations` is stable (or doomed to not receive more work because of Swift) and ship a 1.0.0 version?

Not a blocker for this PR, of course, more of something that popped to mind when seeing `~> 0.0.4`.

Reply:
Agreed. Should be fine to release a 1.0.0. 👍